web-react-ts/src/pages/directory/display/DetailsBacenta.tsx

```diff
@@ -17,7 +17,7 @@ const DetailsBacenta = () => {
   const bacenta = data?.bacentas[0]
 
   let breadcrumb = [
-    bacenta?.constituency.council,
+    bacenta?.constituency?.council,
```

Comment:
How would there be a constituency without a council? 
Are you able to understand how that would happen?

Reply:
I think the data in use query is undefined  on the first render cycle